src/java.base/share/classes/sun/security/util/KeyUtil.java

```diff
@@ -445,5 +448,23 @@ public static boolean isSupportedKeyAgreementOutputAlgorithm(String alg) {
         return alg.equalsIgnoreCase("TlsPremasterSecret")
                 || alg.equalsIgnoreCase("Generic");
     }
+
```

Comment:
As you know, I've been working on the [TLS Exporters change]( https://github.com/openjdk/jdk/pull/24976) which will use the same KDF APIs.  I've already updated that to use your style.  

Looks like I've now got one more thing to change!  ;)

Reply:
This is a reply to the comment above. I don't know why GitHub does not show a reply box there.

> Is any specific salt needed here like in TLS?

In DHKEM, the salt used is always empty.

Reply:
So, no need to explicitly set it I assume? As this is a refactoring, I prefer to minimize changes unless consensus is different.